transformer/app.js

```diff
@@ -65,23 +65,21 @@ const getVulnerablePackages = (infectedFiles) => infectedFiles.map((infectedFile
 }));
 
 const getVulnerabilities = (prefix, impactedArtifact) => ({
-  Id: prefix,
+  Id: prefix.length > 128 ? `${(prefix).substring(0, 125)}...` : prefix,
   VulnerablePackages: getVulnerablePackages(impactedArtifact.infected_files),
 });
 
 const getVulnerabilitiesFields = (prefix, artifact) => ({
   Vulnerabilities: [getVulnerabilities(prefix, artifact)],
 });
 
-const getSummarySubstring = (body) => {
-  return body.summary.length > 256 ? (body.summary).substring(0, 125) + '...' : body.summary;
-}
+const getSummarySubstring = (body) => (body.summary.length > 256 ? `${(body.summary).substring(0, 125)}...` : body.summary);
 
 const getCommonFields = (body, type, accountId, xrayArn) => ({
   AwsAccountId: accountId,
   Region: SECURITY_HUB_REGION,
   CreatedAt: body.created,
-  Description: body.description,
+  Description: body.description.length > 1024 ? `${(body.summary).substring(0, 1020)}...` : body.description,
```

Comment:
It's up to you to implement or not, consider implementing a `truncate()` function to do this since there are 2 instances of this.

Reply:
yeah, good idea, refactored.